3rdparty/rnnoise-build/rnnoise-build.pro

```diff
@@ -0,0 +1,68 @@
+include(../../qmake/compiler.pri)
+
+BUILDDIR=$$basename(PWD)
+SOURCEDIR=$$replace(BUILDDIR,-build,-src)
+
+!win32 {
+  VERSION=0 #$$ Fool packaging script
+}
+
+!exists(../$$SOURCEDIR/COPYING) {
+  message("The $$SOURCEDIR/ directory was not found. Please update your submodules (git submodule update --init).")
+  error("Aborting configuration")
+}
+
+TEMPLATE = lib
+CONFIG -= qt
+CONFIG += debug_and_release
+CONFIG += no_include_pwd
+VPATH = ../$$SOURCEDIR
+TARGET = rnnoise
+DEFINES += HAVE_CONFIG_H
+
+!CONFIG(third-party-warnings) {
+  # We ignore warnings in third party builds. We won't actually look
+  # at them and they clutter out our warnings.
+  CONFIG -= warn_on
+  CONFIG += warn_off
+}
+
+QMAKE_CFLAGS -= -fPIE -pie
+
+win32 {
+  DEFINES += WIN32 _WIN32
+  INCLUDEPATH += ../$$BUILDDIR/win32
+
+  CONFIG -= static
+  CONFIG += shared
+
+  CONFIG(sse2) {
+    TARGET_VERSION_EXT = .sse2
+  } else {
+    QMAKE_CFLAGS_RELEASE -= -arch:SSE
+    QMAKE_CFLAGS_DEBUG -= -arch:SSE
+  }
+}
+
+unix {
+  #CONFIG += staticlib
+
+  contains(QMAKE_CFLAGS, -ffast-math) {
+    DEFINES += FLOAT_APPROX
+  }
+
+  INCLUDEPATH += ../$$BUILDDIR
```

Comment:
Move this out of the `unix` block, so that the path is included for Windows too.